data/data/azure/bootstrap/main.tf

```diff
@@ -188,7 +188,7 @@ resource "azurerm_linux_virtual_machine" "bootstrap" {
   location              = var.azure_region
   resource_group_name   = var.resource_group_name
   network_interface_ids = [azurerm_network_interface.bootstrap.id]
-  size                  = var.azure_bootstrap_vm_type
+  size                  = var.azure_master_vm_type
```

Comment:
We should do one of the following.
1) Change the value set for the `azure_bootstrap_vm_type` variable and use that variable.
1) Or, remove the `azure_bootstrap_vm_type` variable.

If we do (2), then we need to make changes to the azurestack tf files, too.